### DISCUSSION TITLE
Welcome to Undetected Discussions! Read this first if you're new

### DISCUSSION THREAD

**Discussion by ultrafunkamsterdam** [ultrafunkamsterdam/undetected-chromedriver#1058] [Announcements]
#### Reason: THE ISSUE tracker is NOT A SUPPORT FORUM, but at the moment is used like one.  
I will put limitations on  the issue tracker, yet have to find out to which extent.
 
On purpose i allowed such usage, as there are many useful and detailed responses for people to learn,
given that one CAN find them, or WILL search for it.
 
When i recently got **10** new issues **IN A ROW**, regarding the **exact same thing** , I concluded that the above was wishful thinking from my side, and RTFM / FYOI principle is still not evolved.
 
 
We’re using discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Don't demand answers, or even expect one.
  * Share new insights regarding detection advancements.
  * Engage with other community members.
  * Help each other. 

  * Don't advertise! : (real example!)  "UC Detected by XYZ.inc Antibot/WAF startup." 
     when digging some further i found that same dude's name in the source code the company used.
  
  ## Only use the issue tracker for REAL bugs ##
  **which comes down (marginally) to:**
 
not being able to successfully execute this code:
  ```python
import logging
logging.basicConfig(level=10)
import undetected_chromedriver as uc
driver = uc.Chrome()
driver.get('https://nowsecure.nl') # or any other website
```
Or related bugs in the feature set.

**exceptions on this rule for experienced developers and library maintainers using UC one way or another in their (mature) libraries such as SeleniumWire / SeleniumBase and some others**

**Comment by HMaker** [comment]
@ultrafunkamsterdam you reverted this change? the issues are open again?

**Comment by chibuikeogodo** [comment]
Hello, I noticed UC can't allow custom user agent even the latest version, please any hope?

**Reply by cmdlinebeep** [reply]
my god..
